.changeset/polite-dingos-train.md

```diff
@@ -0,0 +1,25 @@
+---
+'@backstage/plugin-kubernetes': minor
+---
+
+Added Pod logs components for Kubernetes plugin
+
+**BREAKING**: `kubernetesProxyApi` must be defined as an apis in the plugin to use k8s proxy features
```

Comment:
This isn't true right? There's a factory installed in the plugin so an instance for this will never be missing.

There's plenty of other breaking changes related to components though